src/tools/rbd_mirror/ImageReplayer.h

```diff
@@ -101,9 +101,10 @@ class ImageReplayer {
     return m_global_image_id;
   }
 
-  void start(Context *on_finish = nullptr, bool manual = false);
+  void start(Context *on_finish = nullptr, bool manual = false,
+             bool restart = false);
   void stop(Context *on_finish = nullptr, bool manual = false,
-	    int r = 0, const std::string& desc = "");
+	    int r = 0, const std::string& desc = "", bool restart = false);
```

Comment:
Nit: does anybody use the `r` and `desc` parameters? If not, can we drop them?

Reply:
Thanks. Updated.